postgres/postgres.go

```diff
@@ -81,15 +97,65 @@ func (ds Datastore) AddJob(host string, newjob *models.Job) (models.Job, error)
 	return *newjob, nil
 }
 
-// Add an uploaded file to a job
+// GetJobs returns a list of import jobs
+func (ds Datastore) GetJobs(host string, filter []string) ([]models.Job, error) {
+	if len(filter) == 0 {
+		filter = allFilterStates
+	}
+	rows, err := ds.getJobs.Query(pg.Array(filter))
+	if err !=nil {
+		return []models.Job{}, err
+	}
+
+	jobs := []models.Job{}
+	for rows.Next() {
+		var jobID, instanceID, jobInfo sql.NullString
+		err = rows.Scan(&jobID, &instanceID, &jobInfo)
+		if err != nil {
```

Comment:
Consider just logging the error and adding a continue here. Then we can try and get the rest of the rows?